index.js

```diff
@@ -105,3 +106,27 @@ exports.stake = async function(options) {
     const result = await account.stake(options.publicKey, BigInt(options.amount));
     console.log('Result: ', JSON.stringify(result));
 }
+
+exports.login = async function(options) {
+    if (!options.walletUrl) {
+        console.log("Log in is not needed on this environment. Please use appropriate master account for shell operations.")
+    } else {
+        const newUrl = new URL(options.walletUrl + "/login/");
+        const title = 'NEAR Shell';
+        newUrl.searchParams.set('title', title);
+        const keyPair = await KeyPair.fromRandom('ed25519');
+        newUrl.searchParams.set('public_key', keyPair.getPublicKey());
+        console.log(`Please navigate to this url and follow the insturctions to log in: ${newUrl.toString()}`);
```

Comment:
s/insturctions/instructions/

probably good idea to put newline before URL to make it easier to copy if not clickable in given terminal